docs/index.rst

```diff
@@ -0,0 +1,18 @@
+Welcome to omero-portlet's documentation!
+==========================================
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   readme
+   installation
+   usage
+   authors
```

Comment:
```suggestion
  
```

Reply:
As this is part of the QUBE template structure, we should make the change their and create a new template release.